compiler/src/iree/compiler/Dialect/Flow/IR/FlowOps.cpp

```diff
@@ -694,6 +694,32 @@ LogicalResult DispatchWorkgroupsOp::verify() {
   return success();
 }
 
+SmallVector<BlockArgument> DispatchWorkgroupsOp::getOutputBlockArguments() {
```

Comment:
Instead of returning all (if there is a use case for it), could we just return the block argument for a single result (based on result number). That seems more lightweight.

Reply:
The use case that I had in mind is passing the entire BlockArguments list to a `bvm.map`, which would be useful when cloning ops into a newly generated WorkgroupsOp. I can add a function for a single result and then this function can call it.

Reply:
There might be other use cases too. Having a function for a single result is more useful. Then maybe even for the `bmv.map` use case it can just iterate and get the values... Not sure though. No strong opinion on this.